src/components/FilterBar.jsx

```diff
@@ -1,11 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Grid, FormControl, Select, MenuItem } from '@mui/material';
+import {
+  Grid,
+  FormControl,
+  Select,
+  MenuItem,
+  Button,
+  Popover,
+} from '@mui/material';
 import AmountFilter from './AmountFilter';
 import GradeLevelFilter from './GradeLevelFilter';
 import qParams from '../lib/QueryParams';
 import sortOptions, { DEADLINE_ASC } from '../lib/sortOptions';
 import MajorFilter from './MajorFilter';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+const Filter = ({ title, filter }) => {
```

Comment:
Maybe `FilterButton`? Since it isn't the same type of component as other `*Filter` ones.